model/src/main/java/org/projectnessie/api/params/EntriesParams.java

```diff
@@ -50,6 +50,12 @@
   @QueryParam("pageToken")
   private String pageToken;
 
+  @Parameter(
+      description =
+          "If set > 0 will filter the results to only return namespaces/tables to the depth of namespaceDepth. If not set or <=0 has no effect")
```

Comment:
nit: Should we also mention that paging is not supported if depth is set?

Reply:
yes, added